index.html

```diff
@@ -1022,6 +1022,12 @@ <h3>Travel (48 countries overall)</h3>
 						<!-- Desktop has 4 columns of 3 units;
 									Mobile has 3 columns of 4 units -->
 						<div class="3u 4u(mobile)">
+
+							<h4>2024</h4>
+							<ul class="countrieslist">
+								<li>Finland</li>

```

Comment:
since this was my first time going to Finland could you please add the new country tag?